rsocket-core/src/commonMain/kotlin/io/rsocket/kotlin/keepalive/KeepAliveHandler.kt

```diff
@@ -21,36 +21,34 @@ import io.rsocket.kotlin.*
 import io.rsocket.kotlin.frame.*
 import kotlinx.atomicfu.*
 import kotlinx.coroutines.*
-import kotlin.time.*
-import kotlin.time.TimeSource.*
 
-@OptIn(ExperimentalTime::class)
 internal class KeepAliveHandler(
     private val keepAlive: KeepAlive,
     private val offerFrame: (frame: Frame) -> Unit,
 ) {
 
-    private val lastMark = atomic<TimeMark?>(null)
+    private val lastMark = atomic(currentMillis())
```

Comment:
Can we use Kotlin TimeSource here?

Reply:
it was, before, but it's still experimental API, in kotlin 1.5 f.e. duration APIs changed in non backward compatible way.
After it will become stable, yes we can.